src/h5web/visualizations/index.test.ts

```diff
@@ -198,48 +197,7 @@ describe('Visualization definitions', () => {
       expect(supportsEntity(group)).toBe(false);
     });
 
-    it('should support group with relative `default` path to supported NXdata group', () => {
-      const group = makeNxGroup('foo', 'NXentry', {
-        defaultPath: 'bar',
-        children: [makeNxDataGroup('bar', { signal: spectrumDatasetInt1D })],
-      });
-
-      expect(supportsEntity(group)).toBe(true);
-    });
-
-    it('should support group with absolute `default` path to supported NXdata group', () => {
-      const rootGroup = makeNxGroup('root', 'NXroot', {
-        children: [
-          makeNxGroup('foo', 'NXentry', {
-            defaultPath: '/foo/bar',
-            children: [
-              makeNxDataGroup('bar', { signal: spectrumDatasetInt1D }),
-            ],
-          }),
-        ],
-      });
-
-      const [groupWithDefault] = rootGroup.children;
-      expect(supportsEntity(groupWithDefault)).toBe(true);
-    });
-
-    it('should support group with multi-step `default` path to supported NXdata group', () => {
-      const group = makeNxGroup('root', 'NXroot', {
-        defaultPath: 'foo',
-        children: [
-          makeNxGroup('foo', 'NXentry', {
-            defaultPath: 'bar',
-            children: [
-              makeNxDataGroup('bar', { signal: spectrumDatasetInt1D }),
-            ],
-          }),
-        ],
-      });
-
-      expect(supportsEntity(group)).toBe(true);
-    });
-
-    it('should not support group with no `NXdata` class and no `default` attribute', () => {
+    it('should not support group with no `NXdata` class', () => {
```

Comment:
The `supportsEntity` tests no longer need to worry about default paths. I've removed/replaced them, as explained earlier.